rocketry/tasks/func.py

```diff
@@ -130,29 +130,30 @@ def my_task_func():
     """
     func: Optional[Callable] = Field(description="Executed function")
 
-    path: Optional[Path] = Field(description="Path to the script that is executed")
+    path: Optional[Path] = Field(description="Path to the script that is executed", default = None)
     func_name: Optional[str] = Field(default="main", description="Name of the function in given path. Pass path as well")
     cache: bool = False
 
     sys_paths: List[Path] = []
 
     _is_delayed: bool = PrivateAttr(default=False)
-    _delayed_kwargs: dict = {}
+    _delayed_kwargs: ClassVar[dict] = {}
```

Comment:
I think these shouldn't be class vars. If I remember correctly, these instance variables when we don't know the function yet.

Reply:
I think this was just sloppy from me. Removing ClassVar here made no difference to tests. This will be changed back to just a dict type on next commit.